starter-web-servlet/src/test/groovy/io/micronaut/starter/servlet/OpenAPISpec.groovy

```diff
@@ -18,19 +19,42 @@ class OpenAPISpec extends Specification {
     @Inject
     HttpClient client
 
-    void "test swagger-ui"() {
+    @Ignore
+    void "test swagger-ui links"() {
```

Comment:
@altro3 has the html output changed for swagger-ui and rapid? These tests which verified the links fail now.

Reply:
Yes, links were changed. Chanes here: https://github.com/micronaut-projects/micronaut-openapi/pull/1626

Currently, links to scripts are not explicitly specified, and scripts are loaded dynamically. The code in the test must be changed